x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/single_search_after.test.ts

```diff
@@ -26,12 +25,13 @@ describe('singleSearchAfter', () => {
 
   test('if singleSearchAfter works without a given sort id', async () => {
     let searchAfterSortId;
-    const sampleParams = sampleRuleAlertParams();
     mockService.callCluster.mockReturnValue(sampleDocSearchResultsNoSortId);
     await expect(
       singleSearchAfter({
         searchAfterSortId,
-        ruleParams: sampleParams,
+        index: [],
+        from: 'now-360s',
+        to: 'now',
```

Comment:
Just a FYI, only the UI converts things to seconds. You can make tests and stuff more readable by using:
```ts
from: 'now-6m'
```

No changes needed, just mentioning. It's simpler for tests.